PROMPTS.md

```diff
@@ -0,0 +1,141 @@
+This is how napari discovers plugins on launch.
+
+## short_description
+
+This should be a short description of what your plugin does. It will be listed
+in `setup.cfg` under the `description` field. If you publish your plugin to PyPI,
+this description will also be listed alongside your package name in search results.
+
+## include_reader_plugin
+
+Choosing `"y"` for this prompt will create an example reader hook implementation 
+inside your plugin's module in the file `_reader.py`. You can then edit the code in this
+file to achieve the reading functionality you want. For more information on
+readers see the [hook specification reference](https://napari.org/plugins/stable/hook_specifications.html#hook-specifications-reference).
+
+## include_writer_plugin
+
+Choosing `"y"` for this prompt will create an example writer hook implementation
+ inside your plugin's module in the file `_writer.py`. You can then edit the code in this file
+to achieve the writing functionality you want. For more information on writers see the [hook specification reference](https://napari.org/plugins/stable/hook_specifications.html#hook-specifications-reference).
+
+## include_dock_widget_plugin
+
+Choosing `"y"` for this prompt will create an example dock widget hook implementation
+ inside your plugin's module in the file `_dock_widget.py`. You can then edit the code in this file
+to achieve the dock widget functionality you want. For more information on dock widgets see the [hook specification reference](https://napari.org/plugins/stable/hook_specifications.html#hook-specifications-reference).
+
+## include_function_plugin
+
+Choosing `"y"` for this prompt will create an example function hook implementation
+ inside your plugin's module in the file `_function.py`. You can then edit the code in this file
+to achieve the functionality you want. For more information on function implementations see the [hook specification reference](https://napari.org/plugins/stable/hook_specifications.html#hook-specifications-reference).
+
+## use_git_tags_for_versioning
+
+The default for this prompt is `"n"`. If you choose `"n"` for this prompt, you will have to manually manage your version numbers
+when you create new releases of your plugin package. You can do
+this in `setup.cfg` under the `version` field. Choosing
+`"n"` at this prompt will add `version = 0.0.1` to your `setup.cfg`. 
+
+If you plan for users to `pip install` your plugin directly from 
+your GitHub repository by pointing to a release asset e.g. a source code archive,
+your users, you may want to choose this option.
+
+If you choose `"y"` for this prompt, versions will be automatically
+managed by `setuptools_scm` when you make commits to your `git` repository.
+This option typically requires the least effort to manage versioning for your
+package. Note that in order to use this option, you must at the 
+very least run `git init` in your package's root directory.
```

Comment:
````suggestion
The default for this prompt is `"n"`. If you choose `"n"` for this prompt, you
will have to manually manage your version numbers when you create new releases
of your package. You can do this in `setup.cfg` under the `version` field (you
will also need to update the version string wherever else you may have used it
in your package, such as in `__init__.py`). Choosing `"n"` at this prompt will
add `version = 0.0.1` to your `setup.cfg`. 

If you choose `"y"` for this prompt, your package will be set up to have
[`setuptools_scm`](https://github.com/pypa/setuptools_scm) manage versions for
you based on your git tags.  See the
[readme](https://github.com/napari/cookiecutter-napari-plugin#automatic-deployment-and-version-management)
for details.

This option typically requires the least effort to manage versioning for your
package, and will prevent errors with manually managed version strings going out
of sync with your package metadata.  The main downside is that your users will
not be able to install directly from a github release asset, and will need to
have git installed if they want to directly install from a git repository.
(This does *not*, however, affect the standard method of installing with `pip`, or
installing from a pre-packaged wheel file.)

```{note}
In order to use this option, you must run `git init` once in
your package's root directory.
```

````

a couple small suggestions in there

Reply:
see also https://github.com/napari/cookiecutter-napari-plugin/pull/48, after which the "you must run `git init`" can be removed